### ISSUE TITLE
【吐槽】tunnel用的窝囊

### ISSUE THREAD

**Issue by Skuyar** [XX-net/XX-Net#9535]
介绍里说的稳定快速真没体验到，看个油管常年缓冲，想流畅看的话240p不能再多了，1080p你做梦呢，想爆粗，fb打开巨JB慢，加载上传个图片转圈半天，我平时也就用用这几个主要网站了，倒是上草榴挺快的。。。。。。
早期的时候还得时不时切换回免费的net用，比tunnel快你能信
不黑，但是目前就是垃圾

要贴的状态图，uploading了半天没上来，不好意思发不了

**Comment by GoodSake** [comment]
哈哈哈！速度的话appid是最快的，功能的话X-Tunnel也不赖吧。

**Comment by Skuyar** [comment]
看需求吧，我就逛图片视频网站多，所以就要求快，稳

**Comment by whhxmwhh** [comment]
是的。因为谷歌后台需要判断很多的事情哦。比如你访问的网站是否是不允许访问的网站。需要一定的过滤机制。因此速度就大大折扣。我自己架设的就飞快的。

**Comment by Steve789** [comment]
用其他工具试试？shadosocks,v2ray等等，自己买VPS自建自用。
